challenge/src/App.jsx

```diff
@@ -11,6 +11,10 @@ class App extends Component {
     }
     render() {
         return (
+            console.log("I am rendering successfully!"),
+            <div style={{ width: "100%", height: "100%", textAlign: "center" }}>
+                {strings.greeting}
+            </div>
```

Comment:
thanks for merging this in, it saves me time, but this got merged a little weird since now there are two divs with the greeting shown twice

Reply:
Removed the duplicate div in newest commit